blob/options.go

```diff
@@ -0,0 +1,27 @@
+package blob
+
+import "cosmossdk.io/math"
+
+// Option is a function that allows to configure the Blob Service requests.
+type Option func(*Service)
+
+// feeParams contains the information about fee and gasLimit price.
+type feeParams struct {
+	fee    math.Int
+	gasLim uint64
```

Comment:
any reason not to call this `gasLimit` vs `gasLim`?

Reply:
renamed to GasLimit